conformance/results/kong-kubernetes-ingress-controller.yaml

```diff
@@ -0,0 +1,55 @@
+typemeta:
+  kind: ""
+  apiversion: ""
+implementation:
+  organization: kong
+  project: kubernetes-ingress-controller
+  url: github.com/kong/kubernetes-ingress-controller
+  version: "2.10"
+  contact:
+  - '@mlavacca'
+  - '@shaneutt'
+date: "2023-07-17T14:54:07+02:00"
+gatewayapiversion: TODO
+profilereports:
+- name: HTTP
+  core:
+    result: partial
+    summary: |
+      'Very first conformance profile.
+      Conformance tests have been run with https://github.com/kubernetes-sigs/gateway-api/commit/7ded95641899d6fb74e60d4793439683430325e7.
+      Full conformance with Gateway-API v0.7.1 (all 23 tests pass).
+      The 6 skipped tests include cases unreleased and pending release in Gateway-API v0.8.'
```

Comment:
This part confuses me, wouldn't you only run the conformance tests as they were released at v0.7.1?

Reply:
Since conformance profiles machinery has been added after 0.7.1, running the suite implies running a set of tests newer than 0.7.1. Because of this, I've written down this message. Actually, I agree this is not the best solution, I'll manually write in the `GatewayAPIVersion` field the current Gateway API version to clarify the level of support. As I wrote in the other comment, such a field should be automatically set, but the machinery to do so is not in place yet.

Reply:
I've rerun the conformance profile CLI and set the GatewayAPIVersion manually to the correct one.